.thoth.yaml

```diff
@@ -59,10 +37,15 @@ managers:
   - name: version
     configuration:
       # A list of maintainers (GitHub or GitLab accounts) of this repository:
-      maintainers: []
+      maintainers:
```

Comment:
I forgot to mention this before: let's remove the `maintainers` section from `.thoth.yaml`, as this repo already has an OWNERS file and Kebechet will take that list from the `approvers` in that file - so, removing this section from here avoids redundancy and potential inconsistencies